kubetest/main.go

```diff
@@ -480,6 +480,10 @@ func prepareGcp(o *options) error {
 			return fmt.Errorf("fail to set project %s : err %v", p, err)
 		}
 
+		if err := os.Setenv("PROJECT", p); err != nil {
```

Comment:
Yes, that function is temporary. Seems like someone is likely to clean up that method but not be sure what to do about here...